scheduler/cups-driverd.cxx

```diff
@@ -22,7 +22,9 @@
 #include <cups/ppd-private.h>
 #include <ppdc/ppdc.h>
 #include <regex.h>
-
+#include <pthread.h>
+#include <sys/time.h>
+#include <unistd.h>
```

Comment:
@pranjanpr please sort all includes alphabetically once you add a new. First are local project includes (uses double quotes), then system includes (uses <>).